MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -599,6 +602,9 @@ private MediaStream GetMediaStream(bool isAudio, MediaStreamInfo streamInfo, Med
             {
                 stream.Type = MediaStreamType.Subtitle;
                 stream.Codec = NormalizeSubtitleCodec(stream.Codec);
+                stream.localizedUndefined = _localization.GetLocalizedString("Undefined");
```

Comment:
why have this in two places?

Reply:
In fact, I am not sure how the ProbeResultNormalizer from MediaEncoder is used, but it creates subtitle MediaStreams, so I copied the behavior here by precaution. 
The SqliteItemRepository create MediaStreams used in itemdetails and videoosd screens.